package.json

```diff
@@ -5,9 +5,11 @@
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
+    "lint": "standard",
     "lint:fix": "standard --fix",
-    "unit": "tap --100 test/**/*.test.js",
-    "test": "standard && npm run unit && npm run test:typescript",
+    "unit": "tap test/**/*.test.js",
+    "test": "npm run unit && npm run test:typescript",
+    "posttest": "rm test/fjs-generated*.js",
```

Comment:
use rimraf instead rm. In the ci cd pipeline we dont get that issue because I think the github runner has some cygwin package installed but on windows I expect that the posttest npm script throws.